docs/code/MlNetCookBook.md

```diff
@@ -303,6 +303,52 @@ private class InspectedRow
 }
 ```
 
+You can also use the dynamic API to create the equivalent of the previous pipeline. 
+```csharp
+// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+// as a catalog of available operations and as the source of randomness.
+var mlContext = new MLContext();
```

Comment:
>var [](start = 0, length = 3)

Please also add these snippets as tests to `CookbookSamples`: this way we ensure that they compile, and are updated when the API gets updated.

Reply:
yes. already created a new set of tests for dynamic api.

---
In reply to: [231221675](https://github.com/dotnet/machinelearning/pull/1538#discussion_r231221675) [](ancestors = 231221675)